website/docs/docs/collaborate/data-tile.md

```diff
@@ -72,11 +72,49 @@ Follow these steps to set up your data health tile:
 
 If your analytics tool supports iFrames, you can embed the dashboard tile within it. 
 
-#### Tableau example
+### Examples
+The following examples show how to embed the data health tile in Tableau and PowerBI.
+
+<Tabs>
+
+<TabItem value="powerbi" label="PowerBI example">
+
+You can embed the data health tile iFrame in PowerBI using PowerBI Pro Online, Fabric PowerBI, or PowerBI Desktop. 
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/power-bi.png" width="80%" title="Embed data health tile iFrame in PowerBI"/>
+
+Follow these steps to embed the data health tile in PowerBI:
+
+1. Create a dashboard in PowerBI and connect to your database to pull in the data.
+2. Create a new measurement with the iFrame.
+3. In dbt Explorer, navigate to the exposure and expand the [**Embed data health into your dashboard**](/docs/collaborate/data-tile#embed-in-your-dashboard) toggle. 
+4. Go to the **iFrame** tab and copy the iFrame code. Make sure the Metadata Only token is already set up.
+5. In PowerBI, under the **Measure tools** tab, replace your values with the iFrame code in the calculation window.
+
+    ```html
+        Website =
+        "<iframe src='https://1234.metadata.us1.dbt.com/exposure-tile?uniqueId=exposure.jaffle_shop.OrderQualityDashboard&environmentType=staging&environmentId=123456789&token=YOUR_METADATA_TOKEN' title='Exposure status tile' height='400'></iframe>"
+    ```
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/power-bi-measure-tools.png" width="100%" title="In the 'Measure tools' tab, replace your values with the iFrame code."/>
+
+6. PowerBI desktop doesn't have HTML rendering by default, so you need to install an HTML visual from the PowerBI Visuals Store.
+7. To do this, go to **Build visuals** and then **Get more visuals**.
+8. Login with your PowerBI account.
+9. Search for an HTML visual and install it. For example, [HTML content](https://appsource.microsoft.com/en-us/product/power-bi-visuals/WA200001930?tab=Overview).
+10. Drag the metric with the iFrame code into the HTML content widget in PowerBI. This should now display your data health tile.
```

Comment:
is the metric with the iframe code visualized with that calculator-looking icon?